test/mainnet/helpers/SetupVault.sol

```diff
@@ -89,7 +91,9 @@ contract SetupVault is Test, MainnetActors, Etches {
         vault.addStrategy(MC.YNETH, 18);
         vault.addStrategy(MC.YNLSDE, 18);
 
-        vault.setBuffer(MC.BUFFER);                                                                  
+        vault.setBuffer(MC.BUFFER);     
+
+        configureCurveActions(vault);                                                           
```

Comment:
Kindly move this test specific configuration to the curve test so the global Setup is generic and simple